database_config/init.sql

```diff
@@ -0,0 +1,178 @@
+--
+-- PostgreSQL database dump
```

Comment:
I put it inside a directory because later on we might have more than 1 script. (Right now the script is just dumping data but we might need some other scripts later on)